pkgs/top-level/all-packages.nix

```diff
@@ -1043,7 +1043,9 @@ with pkgs;
 
   pferd = callPackage ../tools/misc/pferd {};
 
-  qflipper = callPackage ../tools/misc/qflipper { };
+  qFlipper = libsForQt515.callPackage ../tools/misc/qflipper { };
+
+  qflipper = qFlipper;
```

Comment:
IMO we should rename the package, instead of adding an alias here since non-capitalised names are the norm. Looks a bit better i think. 

Then you should also add the old name to `pkgs/top-level/aliases.nix`, that creates an alias for it as well which is meant to be deleted after a while.

Reply:
I've saved the original naming for backward compatibility. Since the package has name "qFlipper", I find it only appropriate to leave it as it is in the attrset. There are plenty of packages having strange-ish names

Will move to aliases, though, thanks :heart: